conf/app.routes

```diff
@@ -10,8 +10,6 @@ GET        /keep-alive              uk.gov.hmrc.taxenrolmentassignmentfrontend.c
 
 GET        /                                                    uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.AccountCheckController.accountCheck(redirectUrl: String)
 
-#GET        /enrol-pt/enrolment-success-sa-user-id               uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.???
```

Comment:
This is required we need the story to implement it.

Reply:
i dont think we should have commented code in our repos, it should be removed until implemented, we may go live without this journey

Reply:
Should we be going live without this journey? The fact it is commented alerts us to the fact it hasn't been done. Had I not seen it I wouldn't have been alerted to the fact that a story hasn't been created for it.

Reply:
it should have been raised as a story in the first place, it should have never been a comment

Reply:
I never said that it should not have been raised as a story in the first place but for whatever reason a story hadn't been created and only by that comment being there was it picked up the fact it is has been missed. We clearly have differing views on this and are not going to agree so will merge your PR.